cocos/platform/ohos/CCFileUtils-ohos.h

```diff
@@ -0,0 +1,69 @@
+#ifndef __CC_FILEUTILS_OHOS_H__
+#define __CC_FILEUTILS_OHOS_H__
+
+#include "platform/CCPlatformConfig.h"
+#if CC_TARGET_PLATFORM == CC_PLATFORM_OHOS
+
+#include "platform/CCFileUtils.h"
+#include "platform/CCPlatformMacros.h"
+#include "base/ccTypes.h"
+#include <string>
+#include <vector>
+#include <unordered_map>
+#include <memory>
+
+#include "napi/modules/RawFileUtils.h"
+
+NS_CC_BEGIN
+
+class ZipFile;
+
+/**
+ * @addtogroup platform
+ * @{
+ */
+
+//! @brief  Helper class to handle file operations
+class CC_DLL FileUtilsOhos : public FileUtils {
+    friend class FileUtils;
+public:
+    FileUtilsOhos();
+    /**
+     * @js NA
+     * @lua NA
+     */
+    virtual ~FileUtilsOhos();
+
+    static void setassetmanager(NativeResourceManager* a);
```

Comment:
Need to keep the same naming rules.
Here should be `setAssetManager` like the following getter `getAssetManager` function.

Reply:
unnecessary assetmanager， deleted